ocaml/xapi/create_misc.ml

```diff
@@ -387,7 +387,52 @@ let make_packs_info () =
 		in
 		Array.fold_left (fun l fname -> get_pack_details fname @ l) [] packs
 	with _ -> []
-  
+
+let run_command cmd =
```

Comment:
Is this the right design? There is `Helper_process.get_process_output` to avoid doing naive fork/exec from a multi-threaded program. Also: whenever you open a file and an exception is thrown while you process it - how do you guarantee that the file is closed? I don't know enough about the Xapi architecture to know for sure but someone who does (like @jonludlam) should take a look here.